auth-sample.json

```diff
@@ -0,0 +1,8 @@
+{
+	"http-basic": {
+		"connect.advancedcustomfields.com": {
+			"username": "ACF License Key",
+			"password": "SITE_URL"
+		}
```

Comment:
Will this require the site to be "registered" with the license key ahead of time? If so, we should probably mention that in the docs.

Reply:
No, there is no registration that is needed.  It will add the site to the active site list in the account section of the ACF site automatically.